src/Files.App/Utils/Shell/PreviewHandler.cs

```diff
@@ -164,7 +164,7 @@ void SetupHandler(Guid clsid)
 			var iid = IPreviewHandlerIid;
 			var cannotCreate = "Cannot create class " + clsid.ToString() + " as IPreviewHandler.";
 			var cannotCast = "Cannot cast class " + clsid.ToString() + " as IObjectWithSite.";
-			var cannotSetSite = "Cannot set site to the preview handler object.";
+			const string cannotSetSite = "Cannot set site to the preview handler object.";
```

Comment:
Would it make sense to move the constant outside of this method?

Reply:
Yes, I think this would improve readability decreasing the method size.

Reply:
Seeing the code, I think it would be more appropriate to inline this string.